internal/driver/service.go

```diff
@@ -0,0 +1,37 @@
+//
+// Copyright (C) 2022 Intel Corporation
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package driver
+
+import (
+	"github.com/edgexfoundry/device-sdk-go/v2/pkg/service"
+	"github.com/edgexfoundry/go-mod-core-contracts/v2/clients/logger"
+	contract "github.com/edgexfoundry/go-mod-core-contracts/v2/models"
+)
+
+// ServiceWrapper wraps an EdgeX SDK service so it can be easily mocked in tests.
```

Comment:
Please submit an issue on the Device SDK to use interface so you don't have to create this wrapper on every service you want to be able to properly unit test

Reply:
https://github.com/edgexfoundry/device-sdk-go/issues/1168